### PR TITLE
[Discussion] Serviceworker cookbook: make all links to MDN

### LINKED ISSUE #11617: Issue with "Client-side storage": (short summary here please)
MDN URL: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Client-side_web_APIs/Client-side_storage

#### What information was incorrect, unhelpful, or incomplete?
The link to the ServiceWorkers cookbook links to https://serviceworke.rs/ which is not the expected destination.

#### Specific section or headline?
[A service worker example](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Client-side_web_APIs/Client-side_storage#a_service_worker_example) > Responding to further requests

#### What did you expect to see?
Expected to link to the Service Worker cookbook - Possibly https://github.com/mozilla/serviceworker-cookbook ?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/learn/javascript/client-side_web_apis/client-side_storage`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Client-side_web_APIs/Client-side_storage
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/javascript/client-side_web_apis/client-side_storage/index.md
* Last commit: https://github.com/mdn/content/commit/832b5d184a3619e8817805f606937732c50a92c4
* Document last modified: 2021-12-30T18:58:22.000Z

</details>

### PR DESCRIPTION
**Pull request by hamishwillee**
Fixes #11617

The serviceworker cookbook is/was linked in a number of places to https://serviceworke.rs/. That URL is now a for-sale domain.
I fixed this by changing links to point to the source code. However it would be better if this could be runnable, so this might not be the right way.

@wbamberg Thoughts on this. I never saw the original cookbook but it seems to me that it was runnable, and that was a useful thing? Happy for this to be closed if we can host this somewhere?

### CODE REVIEW

File: files/en-us/web/progressive_web_apps/re-engageable_notifications_push/index.md [hunk 1]
```diff
@@ -213,7 +214,8 @@ webPush.setVapidDetails(
 );
```
Comment (hamishwillee) [inline files/en-us/web/progressive_web_apps/re-engageable_notifications_push/index.md:214]: This change was to markup the line 210 above which still refers to https://serviceworke.rs/

**Review by sideshowbarker — APPROVED** [review]

**Comment by sideshowbarker** [comment]
I think this PR is fine, but alternatively maybe we should consider not even referencing the https://github.com/mozilla/serviceworker-cookbook repo at all — for one reason, because that repo itself is prominently displaying a link to https://serviceworke.rs/. And given the comment at https://github.com/mozilla/serviceworker-cookbook/issues/333#issuecomment-1004039414, it doesn’t seem like that’s going to be changing any time soon.

**Comment by hamishwillee** [comment]
@sideshowbarker I'm conflicted. Right now I still think it is probably worth linking to because there are a lot of useful real world examples. Also, even though it refers to the .ru site, it also prominently has instructions on how you can build locally yourself. 

I'm going to merge this because at least it makes the MDN docs less incorrect. Happy to come back and remove the references as a post process if we get strong agreement for removal.

While we're here though, do you have any advice on how we might find a maintainer for that repo?

**Comment by sideshowbarker** [comment]
> While we're here though, do you have any advice on how we might find a maintainer for that repo?

I don’t. But I see @Rumya already commented over at https://github.com/mozilla/serviceworker-cookbook/issues/312#issuecomment-985458561 about the proposal to move/merge the repo contents into the mdn/content repo.

Since the serviceworker-cookbook repo is part of the https://github.com/mozilla org, anybody at all who’s a member of that https://github.com/mozilla org and has the Owner bit set should be able to make any changes they want to that repo.

**Comment by wbamberg** [comment]
> @wbamberg Thoughts on this. I never saw the original cookbook but it seems to me that it was runnable, and that was a useful thing? Happy for this to be closed if we can host this somewhere?

Yeah, this has come up a few times lately: https://github.com/mdn/content/search?q=serviceworke&type=issues

In particular there's https://github.com/mdn/content/pull/10697, which suggests proposing a project to adopt the SW cookbook. I think it's a nontrivial amount of work though - AIUI there's a server-side component needed, and the examples have been unmaintained for years, so whoever takes it on should probably make any essential updates at the same time.

**Comment by hamishwillee** [comment]
Thanks guys. Then I will keep my head down until either OWD or MDN team decide what to do.

**Comment by Rumyra** [comment]
Hey - yes I am following up on this. We're looking to move the repo into the MDN org, so at least we can decide on what to do with it there 👍

**Pull request merged**